scripts/lint

```diff
@@ -13,7 +13,7 @@ Execute project linters.
 "
 }
 
-EC_EXCLUDE="(__pycache__|.git|.coverage|coverage.xml|.*\.egg-info)"
+EC_EXCLUDE="(__pycache__|.git|.coverage|coverage.xml|.*\.egg-info|.mypy_cache)"
```

Comment:
Should we add `*.tif` and `*.tiff` to the exclude list instead of adding it in the `.editorconfig` file?

Reply:
Yeah I think that'd be better just so we don't have to keep the list of things to unset sync'd with the things that are set.